llvm/include/llvm/IR/DataLayout.h

```diff
@@ -342,14 +346,63 @@ class DataLayout {
   SmallVector<unsigned, 8> getNonIntegralAddressSpaces() const {
```

Comment:
This name seems stale given it's including unstable pointers

Reply:
I've changed it to getNonStandardAddressSpaces() - not too important since it's only used in unit tests right now.